library/src/fy_library/mixins/property/optional_property_mixin_model/using_mixin_line_fy.py

```diff
@@ -37,16 +37,19 @@ class OptionalPropertyMixinModel_UsingMixinLine_PropertyMixin(
     def _optional_property_mixin_model(self) -> PropertyMixinModel | None:
         # fy:end <<<===
         flow_property_fy_search = _FLOW_PROPERTY_REGEX.search(self._mixin_line)
-        return (
-            PropertyMixinModel(
-                kind=MixinModelKind.PROPERTY,
-                property_name=PythonEntityName.from_snake_case(
-                    flow_property_fy_search.group("property_name")
-                ),
-                implementation_name=PythonEntityName.from_snake_case(
-                    flow_property_fy_search.group("implementation_name")
+
+        if flow_property_fy_search:
```

Comment:
Better approach is early function exit:
```suggestion
        if flow_property_fy_search is None:
            return None
```

Then, the rest of the function is at the main indentation.

Reply:
Change to early function exit